src/main/java/ch/njol/skript/expressions/ExprItemOwner.java

```diff
@@ -0,0 +1,79 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.Bukkit;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Item;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.UUID;
+
+@Name("Item Owner")
+@Description("The owner of the dropped item.")
+@Examples({
+	"broadcast the item owner of all dropped items"
+})
+@Since("INSERT VERSION")
+public class ExprItemOwner extends SimplePropertyExpression<Entity, OfflinePlayer> {
+
+	static {
+		registerDefault(ExprItemOwner.class, OfflinePlayer.class, "item owner", "entities");
+	}
+
+	@Override
+	public @Nullable OfflinePlayer convert(Entity entity) {
+		if (!(entity instanceof Item item))
+			return null;
+		UUID uuid = item.getOwner();
+		if (uuid == null)
+			return null;
+		return Bukkit.getOfflinePlayer(uuid);
```

Comment:
Does this ever involve a lookup to Mojang servers?

Reply:
I'm not sure wym tbh, I dont know how to check.

Reply:
I believe this one does, since the only similar one is https://jd.papermc.io/paper/1.21.4/org/bukkit/Bukkit.html#getOfflinePlayerIfCached(java.lang.String)

Reply:
Unfortunately that uses String for the name and not uuid

Reply:
> Does this ever involve a lookup to Mojang servers?

It will not to my knowledge. Lookups are for turning names to uuids, so if you already have a uuid it can just create the object straight up.

Reply:
How would I go about this testing?

Reply:
spin up a spigot server, run the code with a uuid of a player who's never joined, see if it hangs for a second or two

Reply:
I personally did not experience any hanging. Tried 5 different people.